app/routes/application.js

```diff
@@ -4,16 +4,16 @@ import ApplicationRouteMixin from 'ember-simple-auth/mixins/application-route-mi
 export default Ember.Route.extend(ApplicationRouteMixin, {
 
   actions: {
-    login () {
-      var lockOptions = {
+    login: function() {
```

Comment:
Do you prefer this way of writing a function within your actions/ routes/ components? I have no preference and am sure I am switching back and forth. From this point on should I stick with this format?